### ISSUE TITLE
Handle large packets

### ISSUE THREAD

**Issue by mborgerson** [mborgerson/nxdk-rdt#5]
We currently throw it all at `netconn_write` which is not able to handle much at a time. We should ideally chunk it out. We also don't check the return value for errors.

**Comment by JayFoxRox** [comment]
I feel that this is something that should be abstracted in a way that we can possibly run posix-like socket code soon. That, however, is an issue in NXDK, not in NXDK-RDT.

**Comment by mborgerson** [comment]
LwIP supports a BSD sockets interface, we should probably use it. That would also enable us to build a test app on Linux side to make sure the communication protocol code is correct (and not actually a problem with PktDrv/LwIP/Hardware).

**Comment by JayFoxRox** [comment]
This is also true for the receiving side.
If I try to write more than ~250 bytes from the client, nxdk-rdt will just choke as the data is split amongst a handful of network packets. This is a serious issue which makes nxdk-rdt unusable.

**Comment by mborgerson** [comment]
Yeah, it's broken in multiple ways. First, we don't even consider packets that are split up (we just take the first one), that's the point of this issue. Second, I suspect there's a flaw (probably more) in the LwIP-Xbox interface code. I tried to get it to handle multiple packets but it just kept giving me errors. Just sucks at the moment. Please make it better :).
